src/scenes/Prediction/store.js

```diff
@@ -78,16 +78,22 @@ export default class PredictionStore {
   }
 
   async fetch(limit = this.limit, skip = this.skip) {
-    // if (this.hasMore) {
-    //   const orderBy = { field: 'endTime', direction: this.sortBy };
-    //   const filters = [
-    //     { token: Token.QTUM, status: OracleStatus.VOTING, language: this.app.ui.locale },
-    //     { token: Token.QTUM, status: OracleStatus.CREATED, language: this.app.ui.locale },
-    //   ];
-    //   const { oracles, pageInfo: { hasNextPage } } = await queryAllOracles(this.app, filters, orderBy, limit, skip);
-    //   this.hasMore = hasNextPage;
-    //   return _.orderBy(oracles, ['endTime'], this.sortBy.toLowerCase());
-    // }
+    if (this.hasMore) {
+      const orderBy = { field: 'endTime', direction: this.sortBy };
+      const filters = [
+        { status: EVENT_STATUS.BETTING, language: this.app.ui.locale },
+        { status: EVENT_STATUS.CREATED, language: this.app.ui.locale },
+      ];
+
+      // if naka wallet not loggined, pop up and wait
+      const { account } = this.app.naka;
+      if (!account) await window.ethereum.enable();
```

Comment:
create a convenience method in `NakaStore` and call it from here. this is going to be reused in many places.